docs/context.rst

```diff
@@ -0,0 +1,64 @@
+Context & Breadcrumbs
+=====================
+
+The Java SDK implements the idea of a "context" to support attaching additional
+information to a specific user request, such as breadcrumbs.
+
+Unlike some other platforms, Java has no single definition of what "context" means.
```

Comment:
The comparison to other platforms seems extraneous, and I also don't think this problem is particularly unique to Java. I'd reword this to clarify that the definition of context is dependent on your application (or framework) architecture and leave the rest out of it.